pkg/apply/task/apply_task.go

```diff
@@ -178,6 +178,7 @@ func (a *ApplyTask) Start(taskContext *taskrunner.TaskContext) {
 				taskContext.AddFailedApply(id)
 			} else if info.Object != nil {
 				acc, err := meta.Accessor(info.Object)
+				// TODO: why is apply not successful if Accessor errors?
```

Comment:
I don't know what the right way to handle `err != nil` here, but if it happens now `AddSuccessfulApply` won't get called and neither will `AddFailedApply`...